src/main/java/com/devonfw/ide/sonarqube/common/impl/check/component/DevonArchitectureComponentDeclarationCheck.java

```diff
@@ -15,9 +14,10 @@
  * {@link DevonArchitectureComponentCheck} verifying that a {@link Component} itself is properly defined in
  * {@link Architecture} and that the general {@link Component#getDependencies() dependencies} are not violated.
  */
-@Rule(key = "Devon4j:C1", name = "Devon Component Declaration Check", //
-    description = "Verify component is properly declared in architecture.json file.", //
-    priority = Priority.CRITICAL, tags = { "architecture-violation", "devonfw", "component" })
+// @Rule(key = "Devon4j:C1", name = "Devon Component Declaration Check", //
+// description = "Verify component is properly declared in architecture.json file.", //
+// priority = Priority.CRITICAL, tags = { "architecture-violation", "devonfw", "component" })
```

Comment:
Clean code. Please stop the habbit of committing outcommented code. We have the git history if we need to look up something from the past.